.github/workflows/build-yocto.yml

```diff
@@ -76,8 +76,9 @@ jobs:
 
       - name: Kas build
         run: |
-          export DL_DIR=/srv/gh-runners/quic-yocto/downloads
-          export SSTATE_DIR=/srv/gh-runners/quic-yocto/sstate-cache
+          export PERSISTENT_DIR=/srv/gh-runners/quic-yocto
```

Comment:
I don't think you need PERSISTENT_DIR anymore. it was used before we agreed to use github artifacts for the lock files, no?

Reply:
it is not strictly necessary but simplify the variables definition and is there to be used if we need to store something not using the artifacts.